pkg/selinux/setroubleshoot-client.js

```diff
@@ -35,9 +35,10 @@ var busNameFixit = "org.fedoraproject.SetroubleshootFixit";
 var dbusInterfaceFixit = busNameFixit;
 var dbusPathFixit = "/org/fedoraproject/SetroubleshootFixit/object";
 
-client.init = function() {
+client.init = function(capabilitiesChangedCallback) {
```

Comment:
> It only ever changes once (a simple supports() which returns a promise is enough)

Does it? Already we access two different dbus interfaces and we might get more. Worrying about synchronizing that here seems unnecessary, if we can just triggering an update whenever we get more info. The callback implementation seems more robust to me.

> Using D-Bus introspection data for capabilities checking is awkward and makes the code much more complicated than it needs to be. This is only for the transition right? Can't we just make cockpit depend on the new version of setroubleshoot-server?
Do you have a better solution? The "old" interface reflects the state of fedora-23, which we support. The newest one with all the features we use is in fedora-24 testing. Since the new features aren't essential, I wouldn't want to make cockpit incompatible with people who don't use the setroubleshoot copr for bleeding edge compatibility.